wlauto/external/uiauto/src/com/arm/wlauto/uiauto/UxPerfUiAutomation.java

```diff
@@ -17,7 +17,7 @@
 
 import java.util.logging.Logger;
 
-public class UxPerfUiAutomation extends BaseUiAutomation {
+public class UxPerfUiAutomation extends UxPerfApplaunch{
```

Comment:
But this inheritance hierarchy makes no sense. If anything, Shouldn't `UxPerfApplaunch` inherit from `UxPerfUiAutomation`? Applaunch is a specialisation of  UxPerf workloads, not the other way round.

Honestly, I think the may needs to be refactored to use composition rather than inheritance, otherwise, as new workload types are added, we're going to end up with an increasingly convoluted and unmanageable inheritance hierarchy.